### DISCUSSION TITLE
Hemisphere geometry in MEEP

### DISCUSSION THREAD

**Discussion by varunsv24** [NanoComp/meep#2576] [Q&A] [answered]
I need help to create hemispherical geometry in MEEP. Also please help me to create two such hemispheres and then to merge the two structures to create a single sphere. Please help me. It is very urgent!!!

**Comment by smartalecH** [comment]
Meep has a sphere geometry object you can use: https://meep.readthedocs.io/en/latest/Python_User_Interface/#sphere

**Reply by varunsv24** [reply]
yes it is there. Is it possible to create ha hemisphere geometry in MEEP?

**Reply by smartalecH** [reply, marked as answer]
Sure, just add a block object (listed after the sphere in your geometry list) to cover half of it. The block object should have the same material as the background.

**Reply by varunsv24** [reply]
ok, thank you for your response.
